spec/entry_spec.rb

```diff
@@ -3,42 +3,43 @@
 
 RSpec.describe RegistersClient::Entry do
   Entry_rsf_line = "append-entry\tuser\tGB\t2016-04-05T13:23:05Z\tsha-256:6b18693874513ba13da54d61aafa7cad0c8f5573f3431d6f1c04b07ddb27d6bb"
+  Entry_type = :user
```

Comment:
this is weird casing for Ruby. I think it should be `entry_rsf_line` and `entry_type`?

Reply:
They should start uppercase, as defined here: https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Variables_and_Constants#Constants

Reply:
The convention I've seen around constants is to use all caps (e.g. `ENTRY_TYPE`) for a simple constant and `EntryType` for constants that are either module names or classnames.

Besides that, for this particular case, RSpec provides the `let` construct (https://relishapp.com/rspec/rspec-core/v/3-7/docs/helper-methods/let-and-let):

```ruby
let(:entry_rsf_line) { "append-entry\t...." }
let(:entry_type) { :user }
```

Reply:
Or `subject` if that's the intention https://relishapp.com/rspec/rspec-core/v/3-7/docs/subject/explicit-subject